src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The bulldozer started working");

```

Comment:
The message in the `doWork` method should be more informative and indicate that it is a `Bulldozer` that is working. For example, it could be 'Bulldozer is doing its work.'